packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -81,12 +86,12 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
   const [menuHidden, setMenuHidden] = useState<boolean>(true);
   const containerRef = useRef<HTMLDivElement>(null);
   const theme = useTheme();
-  const { strings } = useLocale();
+  const localeStrings = useLocale().strings.participantItem;
```

Comment:
`ParticipantItem` is becoming too complex. Please split it up into multiple (internal) Components.

OK to do as a follow up refactor PR.